### PR TITLE
fix != filter in volume prune

### PR DESCRIPTION
**Pull request by vyasgun**
Signed-off-by: vyasgun <vyasgun20@gmail.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by vrothberg — APPROVED** [review]
Code LGTM, thanks!

Is there an issue for this fix?

@containers/podman-maintainers PTAL

**Comment by vrothberg** [comment]
Maybe rebasing on the latest HEAD will help

**Comment by vyasgun** [comment]
@vrothberg Thanks for the review. I have rebased and updated the PR -- hopefully it will resolve the issue.

**Review by vrothberg — APPROVED** [review]
/lgtm
/hold

**Comment by vrothberg** [comment]
Tests aren´t happy yet

**Comment by vyasgun** [comment]
Yes, it's my tests. Taking a look

File: test/e2e/volume_prune_test.go
Comment (flouthoc) [inline test/e2e/volume_prune_test.go:153]:
```suggestion
		session = podmanTest.Podman([]string{"volume", "create", "--label", "testlabel", "myvol7"})
```

Volume name already takes in step above, either that volume must be removed or this volume gotta have a new name

**Comment by cevich** [comment]
> Maybe something related to CI itself

I found the error in CI, yep just needed to rebase to fix that.

**Review by vrothberg — APPROVED** [review]
/lgtm

**Pull request merged**
